lib/core/domain/user/user_repository.dart

```diff
@@ -2,7 +2,20 @@ import 'package:app/core/domain/user/entities/user.dart';
 import 'package:app/core/failure.dart';
 import 'package:dartz/dartz.dart';
 
+import '../../application/onboarding/onboarding_bloc/onboarding_bloc.dart';
```

Comment:
It's better code looking when import as `package:app/core/.......` instead of this